src/App.tsx

```diff
@@ -1,51 +1,32 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Routes, Route, Navigate } from 'react-router-dom';
+// Components
+import Navigation from './components/Navigaton';
```

Comment:
Typo in the imported component name: 'Navigaton' should be 'Navigation'

Reply:
Fixed